.github/dependabot.yml

```diff
@@ -9,13 +9,3 @@ updates:
     directory: /
     schedule:
       interval: weekly
-
-  - package-ecosystem: docker
-    directory: /frontend/mariner2
-    schedule:
-      interval: weekly
-
-  - package-ecosystem: gomod
-    directory: /
-    schedule:
-      interval: weekly
```

Comment:
for gomod, some things to consider:
- would changing the interval to monthly help?
- not a lot of value in merging these unless before release, can we leave the prs as is and use it as a signal especially for sec fixes as needed?
- we can also filter for patch releases if that helps

Reply:
I've updated this, PTAL.